.travis.yml

```diff
@@ -3,7 +3,6 @@ sudo: required
 
 python:
   - "2.7"
-  - "3.5"
 
 install:
   - pip install pybuilder
```

Comment:
I already added that here https://github.com/lake-lerna/hydra-deploy/blob/master/.travis.yml. I don't know why it is not showing complete file in pull request.